src/html_support_files/odoc.css

```diff
@@ -740,18 +759,18 @@ td.def-doc *:first-child {
 }
 
 .odoc-toc {
-  position: fixed;
-  top: 0px;
-  bottom: 0px;
-  left: 0px;
-  max-width: 30ex;
-  min-width: 26ex;
-  width: 20%;
+  width: 28ex;
   background: var(--toc-background);
   overflow: auto;
   color: var(--toc-color);
   padding-left: 2ex;
   padding-right: 2ex;
+  grid-row: 2;
+  height: fit-content;
+  border: solid 1px var(--border);
+  border-radius: 5px;
+  position:sticky;
```

Comment:
Sticky is interesting but it has a behavior I find confusing:
If the side bar is longer than the screen is high but is smaller than the content, it's not possible to see the bottom of the side bar until we reach the bottom of the page.

Is here a way to make the side bar sticky only if the bottom of it is visible ?
Otherwise, I think I slightly prefer having a unsticky side bar. It's much more intuitive.

Note: We might make the side bar longer in the future.

Reply:
> Is here a way to make the side bar sticky only if the bottom of it is visible ?

I agree that would be nice. It is certainly possible with javascript. I will look for a css only solution.
I think the sticky is important because the bar is use for navigation inside the page itself.

Reply:
Another solution is to have a maximal height for the left bar of the size of height, and add a scroll bar.

However, I do not love any of the solution!
- Sticky, bottom of long nav bar inaccessible unless at the bottom of the page is a "no" for me
- "Custom stickiness", with some js solving the above problem: why not, but not very satisfying
- Not sticky: simple and works, but not the best since we quickly lose the table of content in long pages

Reply:
There's no perfect solution and I think the scrollable side bar is the best compromise of usability and simplicity.